ci-build.ps1

```diff
@@ -1,11 +1,6 @@
-param(
-  [Parameter(Mandatory = $true, Position = 0)]
-  [string]$targetFramework
-)
-
 dotnet clean -c Release
 
 $repositoryUrl = "https://github.com/$env:GITHUB_REPOSITORY"
 
 # Building for a specific framework.
-dotnet build -c Release -f $targetFramework /p:RepositoryUrl=$repositoryUrl
```

Comment:
This will build every target making the builds longer, won't this?

Does the target framework impact the compiler version, isn't it purely an SDK thing?

Reply:
Yep so I had this badly wrong. My understanding was that specifying the target framework moniker causes the underlying MsBuild call to look for and use that specific SDK to compile against. That's obviously not true. 

I've fixed up the project now so we can build against the preview in CI and everything else is unaffected. I got the langversion working properly also so we can use the C#9 features.